scripts/makeRelease.ts

```diff
@@ -0,0 +1,18 @@
+import {cpSync, mkdirSync, readdirSync, rmSync} from "fs";
+import {resolve} from "path";
+import {BINDINGS_FILE} from "../utils";
+
+const PREBUILD_FOLDER = resolve(__dirname, "..", "prebuild");
+const RELEASE_FOLDER = resolve(__dirname, "..", "release");
+
+cpSync(PREBUILD_FOLDER, RELEASE_FOLDER, {recursive: true});
+rmSync(PREBUILD_FOLDER, {recursive: true, force: true});
+mkdirSync(PREBUILD_FOLDER);
+
+const prebuilds = readdirSync(RELEASE_FOLDER);
+
+for (const binding of prebuilds) {
```

Comment:
why do we have to copy binding files from release folder to prebuild folder? note that we've just removed prebuild earlier in this script

also would be nice to drop some comments for the purpose of this script

Reply:
I was a bit hacky when I wrote that a couple nights ago.  I updated it so its more terse and added a comment for what its doing.

I also added detailed information about the publish process to CONTRIBUTING.md.